packages/web/src/app/shared/component-documentation/component-installation/component-installation.component.ts

```diff
@@ -1,22 +1,33 @@
-import { Component, Input, OnInit } from '@angular/core';
+import { CUSTOM_ELEMENTS_SCHEMA, Component, Input, OnInit } from '@angular/core';
 
 import { CopyComponent } from '../../copy/copy.component';
 import { getPackageName } from './getPackageName';
 import ComponentData from 'src/app/doc-pages/components/component-data.interface';
 
+const LANGUAGE_STORAGE_KEY = 'preferredCegLanguage';
```

Comment:
Should this state (and local storage handling) be its own service maybe, so it's shared between here and the CEG?